build/package/scripts/build-java.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -eu
+
+OUTPUT_DIR="docker"
+WORKING_DIR="."
+ROOT_DIR=$(pwd)
+ARTIFACTS_DIR=$ROOT_DIR/.ods/artifacts
+ARTIFACT_PREFIX=
+DEBUG="false"
+GRADLE_ADDITIONAL_TASKS=
+GRADLE_OPTIONS=
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+
+    --working-dir) WORKING_DIR="$2"; shift;;
+    --working-dir=*) WORKING_DIR="${1#*=}";;
+
+    --output-dir) OUTPUT_DIR="$2"; shift;;
+    --output-dir=*) OUTPUT_DIR="${1#*=}";;
+
+    --gradle-additional-tasks) GRADLE_ADDITIONAL_TASKS="$2"; shift;;
+    --gradle-additional-tasks=*) GRADLE_ADDITIONAL_TASKS="${1#*=}";;
+
+    # Gradle project properties ref: https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties
+    # Gradle options ref: https://docs.gradle.org/current/userguide/command_line_interface.html
+    --gradle-options) GRADLE_OPTIONS="$2"; shift;;
+    --gradle-options=*) GRADLE_OPTIONS="${1#*=}";;
+
+  *) echo "Unknown parameter passed: $1"; exit 1;;
+esac; shift; done
+
+if [ "${WORKING_DIR}" != "." ]; then
+  WORKING_DIR="${ROOT_DIR}"
+  ARTIFACT_PREFIX="${WORKING_DIR/\//-}-"
+fi
+
+if [ "${DEBUG}" == "true" ]; then
+  set -x
+fi
+
+export GRADLE_USER_HOME=/home/gradle/.gradle
+echo "Exported env var 'GRADLE_USER_HOME' with value '${GRADLE_USER_HOME}'"
+echo
+cd "${WORKING_DIR}"
+echo "Working on Java/Gradle project in '${WORKING_DIR}'..."
+echo
+echo "Gradlew version: "
+./gradlew -version
```

Comment:
Maybe we can also print the JDK version?

Reply:
yes, we can do it, but not sure if necessary.
`gradlew -version` actually outputs the JVM version that gradle resolves to use. Other details are displayed too. Here an example:
``` 
Gradlew version: 

------------------------------------------------------------
Gradle 7.2
------------------------------------------------------------

Build time:   2021-08-17 09:59:03 UTC
Revision:     a773786b58bb28710e3dc96c4d1a7063628952ad

Kotlin:       1.5.21
Groovy:       3.0.8
Ant:          Apache Ant(TM) version 1.10.9 compiled on September 27 2020
JVM:          11.0.11 (Red Hat, Inc. 11.0.11+9-LTS)
OS:           Linux 5.4.0-81-generic amd64
``` 
Not sure, should we anyway add `java -version`?

Reply:
Cool. I'd say this is fine then.

BTW: We talked about whether to use `gradlew` or just `gradle` in the CI pipeline - maybe we can open an issue as a follow-up?

Reply:
sounds good, I'll open an issue to follow up on whethere `gradle`or just `gradlew`

Reply:
Opened #211